questions-encodings.mjs

```diff
@@ -7,6 +7,22 @@ import * as use from '@tensorflow-models/universal-sentence-encoder'
 import fetch from 'node-fetch'
 import * as fs from 'fs'
 
+// purge wiki cache to ensure getting latest update of questions
+// https://www.mediawiki.org/wiki/API:Purge
+// POST version of https://stampy.ai/w/api.php?action=purge&titles=Canonically_answered_questions%7CCanonical_answers%7CCanonical_questions
+const urlPurge = 'https://stampy.ai/w/api.php'
+const params = {
+  action: 'purge',
+  titles: 'Canonically_answered_questions|Canonical_answers|Canonical_questions',
+  format: 'json',
+}
+const options = {
+  method: 'POST',
+  body: JSON.stringify(params),
+  headers: {'Content-Type': 'application/json'},
+}
+await fetch(urlPurge, options)
+
```

Comment:
According to @plexish's notes and documentation, I made a POST call to purge the wiki cache. It doesn't give an error but it's not clear that it really works. The JSON response from fetch seems incomplete.